client/my-sites/people/role-select/index.jsx

```diff
@@ -14,44 +14,38 @@ import FormFieldset from 'calypso/components/forms/form-fieldset';
 import FormLabel from 'calypso/components/forms/form-label';
 import FormRadio from 'calypso/components/forms/form-radio';
 import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
+import getWpcomFollowerRole from 'calypso/lib/get-wpcom-follower-role';
 import QuerySites from 'calypso/components/data/query-sites';
-import { getSite, getWpcomFollowerRole } from 'calypso/state/sites/selectors';
+import { getSite } from 'calypso/state/sites/selectors';
 import { ROLES_LIST } from './constants';
+import isPrivateSite from 'calypso/state/selectors/is-private-site';
 import isSiteWPForTeams from 'calypso/state/selectors/is-site-wpforteams';
 import useSiteRolesQuery from 'calypso/data/site-roles/use-site-roles-query';
 
 import './style.scss';
 
 const RoleSelect = ( props ) => {
+	const translate = useTranslate();
 	const { data } = useSiteRolesQuery( props.siteId );
 	const { isWPForTeamsSite } = props;
 
-	const {
-		site,
-		includeFollower,
-		wpcomFollowerRole,
-		siteId,
-		id,
-		explanation,
-		translate,
-		value,
-	} = props;
+	const { site, includeFollower, siteId, id, explanation, value } = props;
```

Comment:
I'm noticing that the `site` prop is not used, except the `if ( site && data )` check, which could as well be just `if ( data )`.

Reply:
Seems like this will be fixed by your PR https://github.com/Automattic/wp-calypso/pull/53578 👍🏻